Open_Source_Events/Cross-WOC/README.md

```diff
@@ -0,0 +1,10 @@
+<h1 align="center"> Cross Winter of Code (CrossWoC) </h1>
+
+## About CrossWoC 
+
+CrossWoC is a six-week-long opensource event organised by IEEE DTU & IEEE DTU CS, which allows programmers and innovators to bring out their nascent talent and find intriguing solutions to real-world problems. It provides a platform for developers to dig deeper into their grey matter and bring out their latent creativity through open source. It is the first edition of CrossWoC conducted by IEEE DTU & IEEE DTU CS, and it will be held from 13th February 2020 to 28th March 2021.
+
+<p align="center">
+<img align="center" alt="GIF" src="https://github.com/Ayush7614/Rotten-Scripts/blob/master/Open_Source_Events/Cross-WOC/cwoc.png" width="320" height="320" />
```

Comment:
Upload the Image on Imgur and reference it here. We have removed all the Images in the repo now.